tests/openvino/test_quantization.py

```diff
@@ -208,8 +208,12 @@ def test_ov_model_static_quantization_with_auto_dataset(
             ov_model.save_pretrained(tmp_dir)
 
```

Comment:
Not directly related to this PR but the conversion step is getting more complex so adding it here : I'd be in favor of removing the onnx conversion step fallback a least for every model which are composed of multiple subcomponents https://github.com/huggingface/optimum-intel/blob/753f84db6e0966580eb9eaa74a808213be730631/optimum/exporters/openvino/convert.py#L465

Reply:
Yes, I think we can do that in separate PR after some experiments to check that we do not have modes that require fallback